src/main/java/online/partyrun/partyrunmatchingservice/global/sse/exception/InvalidSinksKeyException.java

```diff
@@ -0,0 +1,7 @@
+package online.partyrun.partyrunmatchingservice.global.sse.exception;
+
+public class InvalidSinksKeyException extends IllegalArgumentException {
+    public InvalidSinksKeyException() {
+        super("sinks의 key값이 올바르지 않습니다.");
+    }
```

Comment:
```suggestion
    public InvalidSinksKeyException(String key) {
        super(String.format("sinks의 key %s 값이 올바르지 않습니다.", key));
    }
```

이런 식으로 작성하면 어떤 잘못된 값이 들어왔는지 쉽게 확인할 수 있지 않을까요?

Reply:
반영했습니다!

Reply:
Key가 String이 아니어서 다른 예외 클래스로 변환했습니다